paasta_tools/bounce_lib.py

```diff
@@ -214,7 +213,7 @@ def kill_old_ids(old_ids, client):
             continue
 
 
-def get_happy_tasks(app, service, nerve_ns, min_task_uptime=None, check_haproxy=False):
+def get_happy_tasks(app, service, nerve_ns, system_paasta_config, min_task_uptime=None, check_haproxy=False):
```

Comment:
How would you feel if this function took synapse_port instead of system_paasta config?